dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/ClientIT.java

```diff
@@ -347,112 +347,129 @@ private void assertDepsCommandOutput() {
+        checkCommandForHelp(new String[] { WORKFLOW, CONVERT, ENTRY_2_JSON }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, LAUNCH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, VERSION_TAG }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, UPDATE_WORKFLOW }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, MANUAL_PUBLISH }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, "restub" }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, STAR }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, TEST_PARAMETER }, true);
+        checkCommandForHelp(new String[] { WORKFLOW, VERIFY }, true);
+        checkCommandForHelp(new String[] { WORKFLOW }, false);
+
+        checkCommandForHelp(new String[] { PLUGIN, LIST }, true);
+        checkCommandForHelp(new String[] { PLUGIN, DOWNLOAD }, true);
+        checkCommandForHelp(new String[] { PLUGIN }, false);
+
+        checkCommandForHelp(new String[] { CHECKER }, false);
+        checkCommandForHelp(new String[] { CHECKER, DOWNLOAD }, true);
+        checkCommandForHelp(new String[] { CHECKER, LAUNCH }, true);
+        checkCommandForHelp(new String[] { CHECKER, ADD }, true);
+        checkCommandForHelp(new String[] { CHECKER, UPDATE }, true);
+        checkCommandForHelp(new String[] { CHECKER, UPDATE_VERSION }, true);
+        checkCommandForHelp(new String[] { CHECKER, TEST_PARAMETER }, true);
 
     }
 
-    private void checkCommandForHelp(String[] argv) throws IOException {
+    /**
+     * This method passes all the values in argv and --help if includeHelpFlag is true to Client.main and ensures
+     * that the output contains,
+     * 1) "Usage: dockstore", this ensures that some help page is being displayed
+     * 2) "dockstore" followed by the strings in argv, for example, if argv = {"test", "blue", "green"}, it would ensure
+     * that the output contained the string "dockstore test blue green"
+     *
+     * @param argv the arguments to pass to Client.main, do not pass "--help"
+     * @param includeHelpFlag If true, "--help" will be passed to Client.main after the arguments in argv
+     * @throws IOException
+     */
+    private void checkCommandForHelp(String[] argv, Boolean includeHelpFlag) throws IOException {
         final ArrayList<String> strings = Lists.newArrayList(argv);
+        if (includeHelpFlag) {
+            strings.add(HELP);
+        }
         strings.add(CONFIG);
         strings.add(TestUtility.getConfigFileLocation(true));
 
         Client.main(strings.toArray(new String[0]));
-        assertTrue(systemOutRule.getText().contains("Usage: dockstore"));
+        assertTrue(systemOutRule.getText().contains("Usage: dockstore"), "'Usage: dockstore' isn't being displayed, this probably means"
+                + "that no help page is being shown");
+        assertTrue(systemOutRule.getText().contains("dockstore " + join(" ", argv)), "`Usage: dockstore` is being displayed"
```

Comment:
Doing two separate checks, as some help pages look like this,
```
fhembroff@LL6496-Fhembr:~/Documents/work/incorrect-help-pages/dockstore-cli$ dockstore tool convert cwl2yaml --help
 ____             _        _                 
|  _ \  ___   ___| | _____| |_ ___  _ __ ___ 
| | | |/ _ \ / __| |/ / __| __/ _ \| '__/ _ \
| |_| | (_) | (__|   <\__ \ || (_) | | |  __/
|____/ \___/ \___|_|\_\___/\__\___/|_|  \___|

---------------------------------------------
See https://www.dockstore.org for more information

Usage: dockstore tool convert --help
       dockstore tool convert cwl2json [parameters]
       dockstore tool convert cwl2yaml [parameters]
       dockstore tool convert wdl2json [parameters]
       dockstore tool convert entry2json [parameters]

Description:
  They allow you to convert between file representations.

---------------------------------------------


```

Reply:
And although it displays the correct usage for the help page I requested, it is not on the line that begins with `Usage: `.